keps/0004-add-testing-infrastructure.md

```diff
@@ -68,21 +81,62 @@ This is a two phase proposal.
 
 ### Tests
 
-Add testing packages to:
+KUDO will be tested by several different test suites that validate individual components, KUDO, and frameworks.
+
+#### Unit tests
+
+Unit tests test a single component and require no network connectivity. These should be fast, run on every pull request,
+and test that individual KUDO components behave correctly. Unit tests are written as standard Go tests.
+
+All packages should incorporate unit tests that validate their behavior. Some examples of tests that should be included are:
```

Comment:
can we reformat of the text in this doc?   There shouldn't be linefeeds managed by the editor for instance line 88 and 89 should be the same line.

Reply:
I wonder if we could have a linter target for this as it bit me last time too.